seisgo/noise.py

```diff
@@ -679,24 +477,17 @@ def do_correlation(sfile,win_len,step,maxlag,channel_pairs,cc_method='xcorr',aco
             print('source: {}'.format(issta))
             
             # check if the station has assembled raw data
-            if issta in data_cache:
+            if issta in list(data_cache.keys()):
```

Comment:
The way I used did the same thing. 'If issta in data_cache' checks if 'issta' is a key of 'data_cache'.

Reply:
good to know that.